frontend/src/devicelist/DevicesDataTable.jsx

```diff
@@ -0,0 +1,142 @@
+import React from "react";
+import { Box, TableContainer, Typography } from "@material-ui/core";
+
+import {
+  FilterList,
+  Search,
+  ExpandLess,
+  ExpandMore,
+  ArrowDownward,
+  Clear,
+  SaveAlt
+} from "@material-ui/icons";
+
+import MaterialTable, { MTableToolbar } from "material-table";
+
+import ChannelDetailsTable from "./ChannelDetailsTable";
+import StatusIndicator from "../general/StatusIndicator";
+import ActionMenu from "./ActionMenu";
+
+export default class DevicesDataTable extends React.Component {
```

Comment:
I think we can keep the old name `DevicesTable` here. Or even `DeviceTable`.

Reply:
Good point, I could do that. I initially named it differently because I didn't want to delete the old file/table just yet, but since that's been deleted I could rename it as `DeviceTable`.